tests/tests.go

```diff
@@ -2,17 +2,16 @@ package tests
 
 import (
 	"fmt"
-	"math"
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/util/rand"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Is it alright to format this imports? last time I did I have been told to revert in order to prevent diff problems

Reply:
The issue is when you reformat a file that you don't touch otherwise. That may cause collisions. It is fine (and sometimes unavoidable) to format a file that you are changing as a part of the PR.